velox/functions/prestosql/aggregates/tests/PrestoHasherTest.cpp

```diff
@@ -177,12 +177,35 @@ TEST_F(PrestoHasherTest, date) {
 
 TEST_F(PrestoHasherTest, unscaledShortDecimal) {
   assertHash<int64_t>(
-      {0, 1000, std::nullopt}, {0, 2343331593029422743, 0}, DECIMAL(10, 5));
+      {0,
+       1000,
+       -1000,
+       std::nullopt,
+       DecimalUtil::kShortDecimalMax,
+       DecimalUtil::kShortDecimalMin},
+      {0, 1000, -1000, 0, 999999999999999999, -999999999999999999},
```

Comment:
We can use `DecimalUtil::kShortDecimalMax`, `DecimalUtil::kShortDecimalMin` in the expected results as well.

Reply:
Done